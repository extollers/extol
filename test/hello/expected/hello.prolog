% Generated by extoltoprolog
:-(set_prolog_flag(singleton_warning,off)).
:-(discontiguous(/(test,1))).
:-(initialization(;(main,','(write('error: unexpected failure in main'),','(nl,halt(1)))))).
:-(main,','(write('Hello, world!'),','(nl,halt))).
