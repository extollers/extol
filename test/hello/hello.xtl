pred main: ():
  write('Hello, world!'), nl,
  halt.