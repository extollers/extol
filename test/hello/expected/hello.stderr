test/run: line 24: ./hello: No such file or directory
